src/js/helpers/error-handler.js

```diff
@@ -57,7 +57,14 @@ export class ErrorHandler {
         translationId = 'errors.user-exists'
         break
       case errors.TransactionError:
-        translationId = `transaction-errors.${error.errorResults[0].errorCode}`
+        let errorCode
+        const errorResults = error.errorResults
+        if (errorResults) {
+          errorCode = error.errorResults[0].errorCode
+        } else {
+          errorCode = error._resultCodes.operations[0]
+        }
```

Comment:
1. you extracted an `errorResults` variable, but in line 63 use the original source instead of the extracted variable
2. use `lodash/get`
3. you should try extracting from `error._resultCodes.operations[0]` if extracting from `errorResults` failed